rxconfig/auth.yaml.template

```diff
@@ -32,3 +32,4 @@ private-key: >
   xfdbnvKi3Tu8AqIqoEa4BStXvB61m0KcPcZNEZZPpy1ql7iUuu+1VKnyRsE7gl1b
   KnEM6YMz+TW2igQqp2UKjBrwJQ==
   -----END PRIVATE KEY-----
```

Comment:
a private key in the code? is only an example for the template, right?

Reply:
Yes, it is a dummy certificate for testing purposes only.